twitchio/websocket.py

```diff
@@ -141,11 +156,12 @@ async def _connect(self):
             log.error(f"Websocket connection failure: {e}:: Attempting reconnect in {retry} seconds.")
 
             await asyncio.sleep(retry)
-            return asyncio.create_task(self._connect())
+            await self._connect()
```

Comment:
```suggestion
            return await self._connect()
```
you've removed the return here, which will continue the login process as though the connection was successful